src/endpoints/mods.rs

```diff
@@ -119,6 +121,21 @@ pub async fn create(
         .commit()
         .await
         .or(Err(ApiError::TransactionError))?;
+
+    if dev.verified {
+        send_webhook(
```

Comment:
This can probably be threaded with tokyo::spawn, since the dev shouldn't really wait for the webhook to post, and the request shouldn't fail if the webhook fails to post, check this example out: https://github.com/geode-sdk/server/blob/080f4af2355f27088f7c421034908d6e072b3c9d/src/endpoints/mod_versions.rs#L189 which is used to recalculate downloads for a mod